### DISCUSSION TITLE
Using Custom URL with Kick, Set Multiple Instances, Now Program Can Not be Interacted with and makes multiple instances

### DISCUSSION THREAD

**Discussion by chrisfoose** [baffler/Transparent-Twitch-Chat-Overlay#312] [Q&A]
Is there anyway to delete the settings for the program?  I set it up with a Kick URL and multiple instances and there are two problems.

First, I can't enter settings or interact with the program.  Each option I choose when I right-click the program creates a new instance of the program.

Secondly, there is a dialogue box which alerts me that Kick uses cookies.  Now I have to choose Accept or Decline except when I click either nothing happens.

Please help.

**Comment by ratwinkyface** [comment]
I also can't enter the settings or move where the chat is placed on my screen. I try right clicking, uninstalling and reinstalling, but yeah it won't let me interact with it at all. I'm hoping we're able to find a solution soon. In the meantime I just have to use something else :/
